config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1824,7 +1824,7 @@ presubmits:
         - --
         - --deployment=node
         - --gcp-zone=us-central1-b
-        - '--node-test-args=--feature-gates=SidecarContainers=true --service-feature-gates=SidecarContainers=true --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/home/containerd/usr/local/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd\"]}"'
+        - '--node-test-args=--feature-gates=SidecarContainers=true --service-feature-gates=SidecarContainers=true --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd\"]}"'
```

Comment:
We should not be changeing the cgroup driver in the kubelet. Instead the container runtime can be configured to use the systemd driver.

Reply:
Use the [image-config-systemd.toml](
https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/containerd/image-config-systemd.yaml) on [config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml#L1832](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml#L1832)

Reply:
One of the purposes of this job is to not break the `pull-kubernetes-node-e2e-containerd` (required in k/k CI) after merging the SidecarContainers feature to k/k and I guess it uses the cgroupfs driver. Do we have to update it as well?

Reply:
IMHO, we should be updating the cgroup driver for [`pull-kubernetes-node-e2e-containerd`](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml#L44) also.  

@bobbypage WDYT?

Reply:
@gjkim42 I have raised https://github.com/kubernetes/test-infra/pull/29258 to update the same in the tests.

Reply:
updated it to use systemd cgroup driver.